src/wallet/rpcwallet.cpp

```diff
@@ -2801,11 +2801,15 @@ UniValue bumpfee(const JSONRPCRequest& request)
     CFeeRate nOldFeeRate(nOldFee, txSize);
     CAmount nNewFee;
     CFeeRate nNewFeeRate;
+    CFeeRate walletIncrementalRelayFee = CFeeRate(WALLET_INCREMENTAL_RELAY_FEE);
 
     if (totalFee > 0) {
         CAmount minTotalFee = nOldFeeRate.GetFee(maxNewTxSize) + ::incrementalRelayFee.GetFee(maxNewTxSize);
         if (totalFee < minTotalFee) {
-            throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Invalid totalFee, must be at least %s (oldFee %s + relayFee %s)", FormatMoney(minTotalFee), nOldFeeRate.GetFee(maxNewTxSize), ::incrementalRelayFee.GetFee(maxNewTxSize)));
+            CAmount recommendedTotalFee =  nOldFeeRate.GetFee(maxNewTxSize) + walletIncrementalRelayFee.GetFee(maxNewTxSize);
+            throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Invalid totalFee, must be at least %s (oldFee %s + incrementalFee %s), recommended %s (incrementalFee %s)",
```

Comment:
Another another way to avoid the ambiguity of having different "oldfee" values might be to phrase error message to in terms of fee rates, rather than fees:

```
nNewFee = totalFee;
nNewFeeRate = CFeeRate(totalFee, maxNewTxSize);
CFeeRate minFeeRate = CFeeRate(nOldFeeRate.GetFeePerK() + ::incrementalRelayFee.GetFeePerK());
if (nNewFeeRate < minFeeRate) {
    CAmount recommendedTotalFee = nOldFeeRate.GetFee(maxNewTxSize) + walletIncrementalRelayFee.GetFee(maxNewTxSize);
    throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Insufficient totalFee rate %s %s/kB, must be at least %s %s/kB (old rate %s %s/kB + incremental rate %s %s/kB), recommended %s %s (incremental fee %s %s)",
                                                        FormatMoney(nNewFeeRate.GetFeePerK()), CURRENCY_UNIT,
                                                        FormatMoney(minFeeRate.GetFeePerK()), CURRENCY_UNIT,
                                                        FormatMoney(nOldFeeRate.GetFeePerK()), CURRENCY_UNIT,
                                                        FormatMoney(::incrementalRelayFee.GetFeePerK()), CURRENCY_UNIT,
                                                        FormatMoney(recommendedTotalFee), CURRENCY_UNIT,
                                                        walletIncrementalRelayFee.GetFee(maxNewTxSize)));
}
```

I think this would make more sense anyway, because whatever client code is calling this will probably be deriving totalFee values based on fee rates, and this error message is more explicit about the minimum fee rate required.

(Also think saying insufficient fee is better than saying invalid fee, to be more specific).